app/commands/decidim/votings/admin/create_voting.rb

```diff
@@ -25,12 +25,12 @@ def create_voting
             image: form.image,
             start_date: form.start_date,
             end_date: form.end_date,
-            scope: form.scopes_enabled ? form.scope : nil,
+            scope: form.scope,
             importance: form.importance,
             census_date_limit: form.census_date_limit,
             status: form.status,
-            voting_system: form.voting_system,
-            voting_url: form.voting_url,
+            voting_system: 'nVotes',
```

Comment:
Just to know it, why this value fixed here?

Reply:
Well, it has to be fixed somewhere. Currently we are showing a disabled input that does'nt send any  value, so I think that this was a good place instead of adding a hidden field.

Reply:
Sorry, my doubt was because I thought that it was fixed at form level, not with a disabled input. I think that the best option is doing with a method in the form, since you will not have to change the command if a new system is introduced:

```ruby
def voting_system
  "nVotes"
end
```

What do you think?

Reply:
Another thing, I just noticed that `nVotes` is between single quotes. It would be nice to use Decidim's rubocop settings in this module to keep style coherence (in this case, it checks for double quotes in ruby files). I think @jsperezg is using it at https://github.com/podemos-info/decidim-module-crowdfundings.

Reply:
Ok